components/LoginForm.vue

```diff
@@ -67,10 +81,17 @@ export default Vue.extend({
     ...mapActions('deviceProfile', ['setDeviceProfile']),
     ...mapActions('snackbar', ['pushSnackbarMessage']),
     userLogin() {
+      if (!isEmpty(this.user)) {
+        // If we have a user from the public user selector, set it as login
+        this.login.username = this.user.Name || '';
```

Comment:
Reviewing from phone so perhaps I'm missing something obvious, but what's the purpose of this line? Why are we calling it "Name" when what we use is "Username"? I am confused.

Reply:
`this.login.username` is what is passed to the Nuxt Auth scheme.

`this.user` is a `UserDto` object, which has the user's name as a "Name" property.

So essentially, we're skipping having to manually enter the username, by using the name already provided to us by the API.

Reply:
@MrTimscampi Nice, thanks for the clarification. Shouldn't we use something different than an empty string though? Null or undefined for instance.

Reply:
It's mainly to prevent a Typescript error. In practice, you'll never reach this without a valid UserDto object, which always has a name.